### DISCUSSION TITLE
Can't re-verify because my proof always got rejected "because it doesn't include school name"

### DISCUSSION THREAD

**Discussion by davidsl4** [community/community#54376] [GitHub Education]
### Hi everyone,

I tried to reverify my education status with a dated letter from the school, with its name, my name, etc.
It always rejects saying it doesn't include the name of the school

**Comment by samus-aran** [comment]
Hi @davidsl4 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
